hasher-matcher-actioner/hmalib/common.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import logging
+
+
+def get_logger(name=__name__, level=logging.INFO):
```

Comment:
So `__name__` is reserved and refers to the name of what exactly? The calling class?

Reply:
https://docs.python.org/3/reference/import.html?highlight=name#__name__

It's the fully qualified name of the module. eg. `hmalib.lambdas.yada.yada`.